src/components/login/actions.js

```diff
@@ -0,0 +1,41 @@
+import { LOGIN_USER, AUTHENTICATE_USER } from './actionTypes';
+
+/**
+ * Triggers request to authenticate user and log them in using firebase
+ *
+ * @function
+ * @return {Object} The {@link actionTypes.LOGIN_USER LOGIN_USER} action.
+ */
+const loginUser = password => {
+    // convert the password to string for easy slicing and nomenclature
+    const stringPassword = String(password);
+    const stringPasswordWithNumber = `+234${stringPassword}`;
```

Comment:
to convert the number to a string and add the `+234` all you need to do is use a string literal
```
const formattedPassword = `+234${password}`;
```